models/kalosm-llama/src/source.rs

```diff
@@ -55,6 +57,17 @@ pub struct LlamaSource {
     pub(crate) cache: kalosm_common::Cache,
 }
 
+/// Errors that can occur when loading the Llama model.
+#[derive(Debug, thiserror::Error)]
+pub enum Error {
```

Comment:
Not sure whether to give this a proper name, or if the namespace (kalosm::llama::source::Error) is enough for that. Clippy will generally complain when duplicating the last module name as the first word of the type name (for good reason), so SourceError doesn't make sense.

Reply:
I generally prefer more specific names like `LlamaLoadError` to make RA auto imports easier

Reply:
Done in https://github.com/floneum/floneum/compare/e0ed5b9656d2fc9415a2521fea68b927e3c819c7..2607f830bdb2e671db126d8518ff81d6984f3a1b

I went with LlamaSourceError